src/common/com/intellij/plugins/haxe/haxelib/HaxelibLibraryCache.java

```diff
@@ -57,8 +58,10 @@ public HaxelibLibraryCache(@NotNull Sdk sdk) {
 
     for (String libName : installedLibs) {
       HaxeLibrary lib = HaxeLibrary.load(this, libName, mySdk);
-      myCache.add(lib);
-      knownLibraries.add(lib.getName());
+      if (null != lib) {
```

Comment:
I thought about your comment a lot.  I decided that the solution you termed as a workaround is best.  We are already using the `haxelib list` command to gather the library names, and to gather the "dev" info at that point would be fine if we didn't have to deal also with the local .haxelib directories (which we currently do not, but need to).  The trouble there is that we have to use two separate haxelib commands to get the local and global libraries.  They can have duplicates, too, so we have to give priority to the local install if they both exist.  I guess we could do it either way, but I don't mind the current code path.

Feel free to tell me why you disagree, if you do.

Reply:
As a hotfix it's a good idea to leave it as it is. But in the further development we need to invent mechanism that will take in account dev-libs, local `.haxelib` storage and global libraries.